.packit.yaml

```
@@ -0,0 +1,39 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: rubygem-foreman-tasks.spec
+
+# add or remove files that should be synced
+files_to_sync:
+    - rubygem-foreman-tasks.spec
+    - foreman-tasks.logrotate
+    - .packit.yaml
+
+# name in upstream package repository or registry (e.g. in PyPI)
+upstream_package_name: foreman-tasks
+# downstream (Fedora) RPM package name
+downstream_package_name: rubygem-foreman-tasks
+
+actions:
+  post-upstream-clone:
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec -O rubygem-foreman-tasks.spec"
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/plugins/rubygem-foreman-tasks/foreman-tasks.logrotate -O foreman-tasks.logrotate"
+  get-current-version: "ruby -I lib/ -r foreman_tasks/version -e 'puts ForemanTasks::VERSION'"
```

You could also consider a more generic approach of using the gemspec:
```suggestion
  get-current-version: "ruby -rrubygems -e 'puts Gem::Specification::load(Dir.glob("*.gemspec").first).version'"
```

That may make it easier if you want to copy it between various gems.

---

Ah nice, I didn't manage to do it like this.

---

(the quoting is off, tho)